apis/r/R/SOMADataFrame.R

```diff
@@ -80,27 +86,29 @@ SOMADataFrame <- R6::R6Class(
           type = field_type,
           nullable = field$nullable,
           ncells = if (field_type == "ASCII") NA_integer_ else 1L,
-          filter_list = tiledb::tiledb_filter_list(c(
-            tiledb_zstd_filter()
-          ))
+          filter_list = tiledb::tiledb_filter_list(
+            tiledb_create_options$attr_filters(field_name)
+          )
         )
       }
 
       # array schema
+      cell_tile_orders <- tiledb_create_options$cell_tile_orders()
       tdb_schema <- tiledb::tiledb_array_schema(
         domain = tiledb::tiledb_domain(tdb_dims),
         attrs = tdb_attrs,
         sparse = TRUE,
-        cell_order = "ROW_MAJOR",
-        tile_order = "ROW_MAJOR",
-        capacity = 100000,
-        # TODO: should be configurable via a global option
+        cell_order = cell_tile_orders["cell_order"],
+        tile_order = cell_tile_orders["tile_order"],
+        capacity = tiledb_create_options$capacity(),
+        # TODO: should be configurable via PlatformConfig
```

Comment:
In terms of Python/R feature parity, note that Python API does allow this to be configured, even though it's not an explicit property of TileDBCreate: https://github.com/single-cell-data/TileDB-SOMA/blob/2f17f258e697de2c6012a4e610acd5a1135fa196/apis/python/src/tiledbsoma/_common_nd_array.py#L180

Reply:
Oh wow thanks @atolopko-czi ! This will be another instance of #1229 -- I'll put up a Python PR for `allow_dups`, imitating what #1229 did for `capacity`.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/1234

Reply:
Also added on the R side on the most recent commit on this PR